src/test/regress/expected/failure_create_table.out

```diff
@@ -0,0 +1,513 @@
+CREATE SCHEMA failure_create_table;
+SET search_path TO 'failure_create_table';
+SELECT citus.mitmproxy('conn.allow()');
```

Comment:
Given that Citus create schemas in a slightly odd way, could you add some failure tests while creating the schema via `EnsureSchemaExistsAllNodes()`?